lib/auth/db.go

```diff
@@ -88,6 +90,17 @@ func (s *Server) GenerateDatabaseCert(ctx context.Context, req *proto.DatabaseCe
 	}, nil
 }
 
+// getCaAndSigner returns correct signer and CA that should be used when generating database certificate.
```

Comment:
`getCAAndSigner`? (comment and function name don't match)

Reply:
Thanks, fixed.